windows_containers/windows-containers-release-notes-8-x.adoc

```diff
@@ -53,6 +53,8 @@ The `must-gather` tool now collects the service logs generated by the Windows In
 
 * Previously, the test to determine if the Windows Defender antivirus service is running was incorrectly checking for any process whose name started with `Windows Defender`, regardless of state. This resulted in an error when the WMCO created firewall exclusions for containerd on instances without `Windows Defender` installed. This fix now checks for the presence of the specific running process associated with the Windows Defender antivirus service. As a result, the WMCO can properly configure Windows instances as nodes regardless of whether Windows Defender is installed. (link:https://issues.redhat.com/browse/OCPBUGS-1513[*OCPBUGS-1513*])
 
+* Previously, in-tree storage was not working for Windows nodes on VMware vSphere. With this fix, {productwinc} properly supports in-tree storage for all cloud providers.  (link:https://issues.redhat.com/browse/WINC-1014[*WINC-1014*])
+
```

Comment:
@sebsoto Can you PTAL at this commit to add your in-tree storage fix to the 8.0.0 release notes?